superbench/benchmarks/directx_benchmarks/DirectXGPUCoreFLOPs/GPUCore.vcxproj

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Shall we merge filters and sln into vcxproj?

Reply:
updated